src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/flyout/flyout.js

```diff
@@ -549,7 +549,7 @@ export class Flyout extends Component {
     const { close } = this.props;
 
     return (
-      <EuiFlyout onClose={close}>
+      <EuiFlyout onClose={close} size="s" maxWidth>
```

Comment:
I guess `maxWidth` isn't needed after all!

Reply:
Could you remove it?

Reply:
By default, the `max-width` class isn't set by `EuiFlyout` (only `width` and `min-width` seem to be set at the resolutions I tested on). I did need to add the `maxWidth` prop.

Reply:
@scottis Is your EUI dependency up-to-date? I just tested this by running `yarn kbn bootstrap` to get my dependencies up-to-date and removed the `maxWidth` prop. Everything looks great in the browser, and this is what I see when I inspect the element:

![image](https://user-images.githubusercontent.com/1238659/44168248-79d69280-a085-11e8-8732-2da8ba5ae97d.png)

Looks like the `min-width` is already built into the `euiFlyout--small` class.